companies/unicorn/index.md

```diff
@@ -10,12 +10,12 @@ tags:
   - startup
   - prop-tech
 company_url: https://www.unicorn.de/
-job_board_url: https://unicornjobs.heavenhr.com/jobs
-job_board_provider: heavenhr
-job_board_hostname: unicornjobs
+job_board_url: https://www.unicorn.de/en/jobs
```

Comment:
it seems that https://unicornjobs.heavenhr.com/jobs is still active

this allows the use of
```
job_board_provider: heavenhr
job_board_hostname: unicornjobs
```

there is also a confusing key naming, between the two urls:
- https://www.unicorn.de/en/jobs
- https://unicornjobs.heavenhr.com/jobs

The current company model does not allow both, maybe it should.